Src/GoogleApis.Auth.DotNet4/OAuth2/ServiceAccountCredential.cs

```diff
@@ -41,6 +41,14 @@ namespace Google.Apis.Auth.OAuth2
     /// <para>
     /// Take a look in https://developers.google.com/accounts/docs/OAuth2ServiceAccount for more details.
     /// </para>
+    /// <para>
+    /// Service account credential also supports JSON Web Token access token scenario.

```

Comment:
You can also add a prefix: From version 1.9.3, ...
So: "From version 1.9.3, service account credential also....